features/gebruik_in_lopende_tekst.feature

```diff
@@ -1,189 +1,727 @@
+    | geslachtsaanduiding | V         |
+    En de persoon heeft de volgende naam gegevens
+    | naam                           | waarde |
+    | voorvoegsel (02.30)            |        |
+    | geslachtsnaam (02.40)          | Wit    |
+    | voornamen (02.10)              | Jan    |
+    | aanduiding naamgebruik (61.10) | V      |
+    En de persoon heeft een ex-partner met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992935 |
+    En de ex-partner heeft de volgende naam gegevens
+    | naam                  | waarde |
+    | voorvoegsel (02.30)   |        |
+    | geslachtsnaam (02.40) | Zwart  |
+    En de ex-partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+    | naam                     | waarde   |
+    | datum aangaan (06.10)    | 19580401 |
+    En de ex-partner heeft de volgende ontbindingHuwelijkPartnerschap gegevens
+    | naam                     | waarde   |
+    | datum ontbinding (07.10) | 20061001 |
+    En de persoon heeft een ex-partner met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992936 |
+    En de ex-partner heeft de volgende naam gegevens
+    | naam                  | waarde |
+    | voorvoegsel (02.30)   |        |
+    | geslachtsnaam (02.40) | Blaauw |
+    En de ex-partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+    | naam                     | waarde   |
+    | datum aangaan (06.10)    | 19610422 |
+    En de ex-partner heeft de volgende ontbindingHuwelijkPartnerschap gegevens
+    | naam                     | waarde   |
+    | datum ontbinding (07.10) | 19831014 |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999992934                       |
+    | fields              | naam.gebruikInLopendeTekst      |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+    | naam                  | waarde            |
+    | gebruikInLopendeTekst | mevrouw Zwart-Wit |
+
+Rule: Wanneer de geslachtsnaam van de persoon leeg of onbekend is en de naam van de persoon wordt gebruikt, wordt gebruikInLopendeTekst niet opgenomen
+
+    Abstract Scenario: naam van de persoon is onbekend bij aanduiding naamgebruik "<naamgebruik>"
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    | geslachtsaanduiding | V          |
+    En de persoon heeft de volgende naam gegevens
+    | naam                                 | waarde        |
+    | voornamen (02.10)                    |               |
+    | adellijke titel of predikaat (02.20) |               |
+    | voorvoegsel (02.30)                  |               |
+    | geslachtsnaam (02.40)                | .             |
```

Comment:
Even een check. De proxy krijgt geen geslachtsnaam ipv een geslachtsnaam property met waarde '.'. Als dit klopt, dan zou ik hier de . als waarde weglaten en zou ik een andere scenario opnemen om te beschrijven dat de GBA API bij een geslachtsnaam (02.40) met waarde . geen geslachtsnaam in de naam retourneert

Reply:
laten we dit in overleg bespreken

Reply:
de RvIG API levert vooralsnog de waarde zoals die in de GBA-V staat, dus voor een onbekende geslachtsnaam zal dat "." zijn. De proxy filtert dat eruit. Voor het bepalen van gebruikInLopendeTekst heeft geslachtsnaam in de bron/input dus een ".".
Maar ik maak hier een Abstract Scenario van met beide situaties (spatie en punt).